ElectronClient/gui/NoteContentPropertiesDialog.tsx

```diff
@@ -42,6 +50,8 @@ export default function NoteContentPropertiesDialog(props:NoteContentPropertiesD
 	const [strippedWords, setStrippedWords] = useState<number>(0);
 	const [strippedCharacters, setStrippedCharacters] = useState<number>(0);
 	const [strippedCharactersNoSpace, setStrippedCharactersNoSpace] = useState<number>(0);
+	const [strippedReadTime, setStrippedReadTime] = useState<number>(0);
+	const words_per_minute = 200;
```

Comment:
What is this number based on? Any source?

Reply:
And please use camel case. In case it's not obvious from the dozens of camel case variables in this file, we use camel case.

Reply:
Whoops, my strong python background pulling through.

Based on lower end of the 200-250 range of a google search for "Average reading speed".